core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -262,21 +267,26 @@ fun LPGenericElevatedCard(
         colors = CardDefaults.cardColors(
             containerColor = color,
         ),
-        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(12.dp)),
+        shape = MaterialTheme.shapes.medium.copy(all = CornerSize(LeasePertTheme.sizes.midSmallSize)),
         modifier = Modifier
-            .width(312.dp)
-            .height(196.dp)
-            .padding(16.dp)
+            .width(LeasePertTheme.sizes.extraSize336)
+            .height(LeasePertTheme.sizes.extraSize199)
+            .padding(LeasePertTheme.sizes.smallSize)
```

Comment:
Please add the name of the properties padding in this case `padding(all = LeasePertTheme.sizes.smallSize)`

Reply:
Working on the comments

Reply:
Comment resolved